Makefile

```diff
@@ -1,14 +1,57 @@
 SHELL := /bin/bash
 
+# helm variables
 IDENTIFIER ?= $(shell date +%s)
 RELEASE_NAME ?= $(shell whoami)-${IDENTIFIER}
 NAMESPACE_NAME ?= $(shell whoami)-${IDENTIFIER}
 NIFI_CHART_DIR ?= nifi
 DEPLOYED_NS ?= ""
 DEPLOYED_RELEASE ?= ${DEPLOYED_NS}
+
 STRUCTURED_LOG ?= false
+MEMORY_GB ?= 50
```

Comment:
Not sure about the intention of introducing request memory in the makefile.
If it is about using a low value then there is a values file, `small-resources-values.yaml` which tackles to deploy the cluster for testing purpose with smaller footprint.

Reply:
that requires that you run helm directly...  trying to make the makefile pass it forward instead and provide overrides